python/tvm/contrib/pipeline_executor.py

```diff
@@ -501,17 +538,18 @@ class PipelineExecutorFactoryModule(object):
 
     """
 
-    def __init__(self, pipeline_mods, mods_config):
-        mods, config = self.graph_executor_create(pipeline_mods, mods_config)
-        assert (
-            pipeline_executor_enabled()
-        ), "Pipeline executor is not enabled. Please \
-              re-build TVM with USE_PIPELINE_EXECUTOR=ON"
-        pipeline_create = tvm._ffi.get_global_func(
+    def __init__(self, pipeline_libs=None, mods_config=None):
+        self.pipeline_libs = pipeline_libs
+        self.mods_config = mods_config
+        self.pipeline_create = tvm._ffi.get_global_func(
```

Comment:
You set `allow_missing=False`, so the program will crash here without a message.

Reply:
Please do not resolve conversation before checked by the commenter.
Now you allow this to be None, but only create graph executor when it is not. Then if this function is None, users will get nothing silently. Please add the proper error message instead of silently doing nothing.

Reply:
change allow_missing into False, then would throw a error when get function failed.